Examples/Tests/photon_photon_collisions/inputs_many_photons_3d

```diff
@@ -0,0 +1,147 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 100
+amr.n_cell = 16 16 16
+amr.max_grid_size = 8
+amr.blocking_factor = 2
+amr.max_level = 0
+geometry.dims = 3
+geometry.prob_lo = -1e-6 -1e-6 -1e-6
+geometry.prob_hi = 1e-6 1e-6 1e-6
+
+#################################
+##### BOUNDARY CONDITIONS #######
+#################################
+boundary.field_lo = periodic periodic periodic
+boundary.field_hi = periodic periodic periodic
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.verbose = 1
+warpx.cfl = 1
+algo.particle_shape = 1
+algo.maxwell_solver = yee
+algo.current_deposition = esirkepov
+algo.field_gathering = energy-conserving
+algo.particle_pusher = boris
+
+#################################
+############ PLASMA #############
+#################################
+particles.species_names = photonA photonB electron positron
+particles.photon_species = photonA photonB
+
+photonA.species_type = photon
+photonA.injection_style = "NRandomPerCell"
+photonA.num_particles_per_cell = 100
+photonA.profile = constant
+photonA.density = 1.e35
+photonA.momentum_distribution_type = "constant"
+photonA.ux = 2.8
+photonA.uy = 0.
+photonA.uz = 0.
+photonA.do_not_push = 1
+photonA.do_not_deposit = 1
+
+photonB.species_type = photon
+photonB.injection_style = "NRandomPerCell"
+photonB.num_particles_per_cell = 100
+photonB.profile = constant
+photonB.density = 1.e35
+photonB.momentum_distribution_type = "constant"
+photonB.ux = -2.8
+photonB.uy = 0.
+photonB.uz = 0.
+photonB.do_not_push = 1
+photonB.do_not_deposit = 1
+
+electron.species_type = electron
+electron.injection_style = none
+electron.do_not_push = 1
+electron.do_not_deposit = 1
+
+positron.species_type = positron
+positron.injection_style = none
+positron.do_not_push = 1
+positron.do_not_deposit = 1
```

Comment:
same as above

Reply:
This is convenient to estimate correctly the electron-positron yield with the analytical formula based on the cross-section. Otherwise the photons would start generating a pair plasma that evolves self-consistently (possibly with instabilities if the density is too high, depending on the parameters) and everything becomes complicated. Same applies to electrons.